src/ucp/core/ucp_worker.inl

```diff
@@ -122,6 +122,13 @@ ucp_worker_extract_request_by_id(ucp_worker_h worker, ucs_ptr_map_key_t id)
     return request;
 }
 
+static UCS_F_ALWAYS_INLINE int
+ucp_worker_keepalive_is_enabled(ucp_worker_h worker)
+{
+    return (worker->context->config.keepalive_interval != 0) &&
+           (worker->context->config.ext.keepalive_num_eps != 0);
```

Comment:
per discussion with @brminich , need to check only worker->context->config.keepalive_interval != 0 here,
and if user sets worker->context->config.ext.keepalive_num_eps to 0, it's considered invalid value and need to fail

Reply:
ok, done